spec/classes/repo_spec.rb

```diff
@@ -426,6 +426,72 @@
     it { should contain_yumrepo('zabbix-nonsupported').with_baseurl('http://repo.zabbix.com/non-supported/rhel/6/$basearch/') }
   end
 
+  context 'on RedHat 6 and Zabbix 3.0' do
+    let :params do
+      {
+        zabbix_version: '3.0',
+        manage_repo: true
+      }
+    end
+
+    let :facts do
+      {
+        :operatingsystem            => 'RedHat',
+        :operatingsystemrelease     => '6',
+        :operatingsystemmajrelease  => '6',
+        :architecture               => 'x86_64',
+        :osfamily                   => 'RedHat',
+        :$majorrelease              => '6',
```

Comment:
please remove the $. is it possible for you to migrate (parts of) this spec file into rspec-puppet-facts? Basic example: https://github.com/voxpupuli/puppet-zabbix/blob/master/spec/classes/userparameter_spec.rb#L7-L15